### PR TITLE
TC includes own invades

### PR DESCRIPTION
**Pull request by stromblom**
Solves #428

### CODE REVIEW

File: src/Http/Controllers/Dominion/TownCrierController.php [hunk 1]
```diff
@@ -15,9 +15,6 @@ public function getIndex()
 
         $realm = $dominion->realm;
         $realmieDominionIds = $realm->dominions
```
Comment (WaveHack) [inline src/Http/Controllers/Dominion/TownCrierController.php]: Can you change this variable to $dominionIds?

Comment (stromblom) [reply src/Http/Controllers/Dominion/TownCrierController.php]: Sure

Comment (WaveHack) [reply src/Http/Controllers/Dominion/TownCrierController.php]: Also in the compact() call below, and in the view

Comment (stromblom) [reply src/Http/Controllers/Dominion/TownCrierController.php]: Solved

**Pull request merged**